config/v1/types_apiserver.go

```diff
@@ -63,6 +66,28 @@ type APIServerNamedServingCert struct {
 	ServingCertificate SecretNameReference `json:"servingCertificate"`
 }
 
+type APIServerEncryption struct {
+	// type defines what encryption type should be used to encrypt resources at the datastore layer.
+	// Once this field is set to a value other than the empty string or identity, it cannot be updated.
+	// For example, setting type to aescbc permanently enables encryption of type aescbc;
+	// the type cannot be changed to a different value at a later time.
+	// When this field is unset (i.e. when it is set to the empty string), identity is implied.
```

Comment:
Indicate that we reserve the option to change the default.  I'd like to see it on by default one release after it first comes out.

Reply:
Updated.